lumicks/pylake/tests/test_import_time.py

```diff
@@ -0,0 +1,23 @@
+from textwrap import dedent
+import numpy as np
+import subprocess
+import sys
+import pytest
+
+
+@pytest.mark.slow
+def test_disabling_capturing(capsys):
+    repeats = 3
+
+    code = dedent("""\
+        import time
+        tic = time.time()
+        import lumicks.pylake
+        print(time.time() - tic)
+        """)
+
+    times = [float(subprocess.check_output([f'{sys.executable}', '-c', code])) for i in range(repeats)]
+
+    with capsys.disabled():
+        sys.stdout.write(f"\n \033[92mImport takes: "
+                         f"{np.mean(times):.2f} +- {np.std(times):.2f} seconds (N={repeats})\033[0m")
```

Comment:
Consider dropping the newline and `Import takes` to better line up the results (the file name `test_import_time` already says what the time is). For example, `f": {mean} +-{std} seconds (N={repeats})"` would produce this:
```
lumicks/pylake/tests/test_image.py ......                                [ 83%]
lumicks/pylake/tests/test_import_time.py: 0.64 +-0.01 seconds (N=3).     [ 84%]
lumicks/pylake/tests/test_kymotracker.py ...................             [ 94%]
```

Reply:
You get this instead:
![image](https://user-images.githubusercontent.com/19836026/98592789-7f59cc00-22d2-11eb-8a13-3f22b36fca77.png)

Reply:
How about abusing atexit? 👼 

![image](https://user-images.githubusercontent.com/19836026/98661059-4ce9b680-2346-11eb-93ce-6540404df686.png)

Reply:
Do it! 🚀

Reply:
I set it up as a fixture now. That way we can have different tests use the same reporting mechanism if this is needed again in the future. Or do you think this is premature and should I just keep it inside that one test for now?

Reply:
Sounds perfect!